hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/util/FutureMessage.java

```diff
@@ -0,0 +1,30 @@
+package org.apache.mesos.hdfs.util;
+
+import org.apache.mesos.Protos;
+import org.apache.mesos.SchedulerDriver;
+import org.apache.mesos.hdfs.scheduler.Scheduler;
+
+/**
```

Comment:
the problem is there is state that needs to be carried into the future... it can't be just a method.  I will assume you want an inner class instead.